File: Elevator/Java/src/main/java/frc/robot/gamepad/OI.java
package frc.robot.gamepad;

import edu.wpi.first.wpilibj.Joystick;

public class OI
{
    //Create the joystick
    Joystick drivePad;

    public OI()
    {
        //initialize the joystick
        drivePad = new Joystick(GamepadConstants.DRIVE_USB_PORT);
    }

    /**
     * @return the y-axis value from the drivePad right joystick
     */
    public double getRightDriveY()
    {
        double joy = drivePad.getRawAxis(GamepadConstants.RIGHT_ANALOG_Y);
        return Math.abs(joy) < 0.05 ? 0.0 : joy;
    }
}

File: Elevator/C++/src/main/cpp/gamepad/OI.cpp
#include "gamepad/OI.h"
#include "frc/Joystick.h"
#include <cmath>

frc::Joystick drivePad(DRIVE_USB_PORT);

/**
 * @return the y-axis value from the drivePad right joystick
 */
double OI::GetRightDriveY(void)
{
    double joy = drivePad.GetRawAxis(RIGHT_ANALOG_Y);
    if(fabs(joy) < 0.05)
        return 0.0;
    else
        return joy;
}